seqr/views/apis/saved_variant_api.py

```diff
@@ -210,6 +210,26 @@ def update_variant_tags_handler(request, variant_guids):
         get_tag_create_data=_get_tag_type_create_data, get_tags_json=get_json_for_variant_tags,
         delete_variants_if_empty=True)
 
+@login_and_policies_required
+def update_variant_acmg_classification_handler(request, variant_guid):
+    return _update_variant_acmg_classification(request, variant_guid)
+
+def _update_variant_acmg_classification(request, variant_guid):
+    request_json = json.loads(request.body)
```

Comment:
you shouldn't be parsing the request data until after you have checked the permission the saved variant